recipe/meta.yaml

```diff
@@ -32,22 +32,25 @@ requirements:
     - python    # [win]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [not win]
```

Comment:
Sorry I misread. Though we do have to do something about that line too.

Unfortunately we need `gcc` on OS X too if we want to have access to `gfortran`. Mac doesn't provide a  `gfortran` compiler. I believe they use to, but that was on 10.4. Fortunately, the work we did getting `clang` to be the compiler and `libc++` used should actually help us here. As a result, we should only need to ship `libgfortran` with this on OS X. I just went through a similar ordeal with `scipy`. So, am pretty confident that this should work. We still want to review this carefully nonetheless.